src/Microsoft.Restier.EntityFramework7/Model/ModelProducer.cs

```diff
@@ -202,6 +203,29 @@ private static void SetComputedAnnotation(EdmModel model, IEdmProperty target)
             {
                 return null;
             }
+
+            if (Type.GetTypeCode(propertyType) == TypeCode.DateTime)
```

Comment:
Oh it's my fault, this is written before I brought TypeHelper in, will change it.